compiler.go

```diff
@@ -91,10 +97,18 @@ type compiler struct {
 	stringCache map[unistring.String]Value
 }
 
+func (c *compiler) getScriptOrModule() interface{} {
+	if c.module != nil {
+		return c.module
+	}
+	return c.p // TODO figure comething better
```

Comment:
```suggestion
	return c.p // TODO figure something better
```
Is it expected to keep it?